### DISCUSSION TITLE
Column sorting

### DISCUSSION THREAD

**Discussion by hbasaraner** [nick-keller/react-datasheet-grid#53] [Feature requests]
Hello. 👋🏻 Is there any property for column sorting? If isn't, will be added in future updates? Thanks.

**Comment by nick-keller** [comment]
Hi @hbasaraner,
Nothing planned on that matter yet, could you pitch your idea?

**Comment by hbasaraner** [comment]
> Hi @hbasaraner,
> Nothing planned on that matter yet, could you pitch your idea?

Hi. I'm just developing an application to show my Excel data. It contains hundreds of data, so it's hard to find what I want exactly. Not a big problem right now, but it might be good for me. By the way, I'm implementing a new column type to your project. I'll be glad if I can make a contribution.

**Comment by nick-keller** [comment]
If you have a precise use-case with an API idea feel free to share, will gladly consider it for future releases ;)

**Comment by otariterterashvili** [comment]
@nick-keller No sorting still ?

**Reply by nick-keller** [reply]
Hello @otariterterashvili , could you pitch your needs?
Let's assume the feature exists, how would you expect it to work? Can you show some code of the potential usage?

**Reply by otariterterashvili** [reply]
Hello @nick-keller, I work to add this functionality in my project. I will share code.

**Reply by alkima5** [reply]
hey is there any progress?

**Reply by nick-keller** [reply]
Hey @alkima5,
No progress. If you would like to see this hapenning could you please share your use-case:
- What should it do?
- How should it work?
- What do you expect the API to look like?
